docs/requirements-docs.txt

```diff
@@ -3,6 +3,7 @@ mkdocs-material
 pymdown-extensions
 markdown
 mkdocstrings
-presidio_analyzer
-presidio_anonymizer
-presidio_image_redactor
\ No newline at end of file
+
+# presidio_analyzer 
```

Comment:
uncommented here so that we could install them locally if needed to generate docs.